README.md

```diff
@@ -1,176 +1,203 @@
+The Astro CLI is a command-line interface for data orchestration. It allows you to get started with Apache Airflow quickly and it can be used with all Astronomer products.
 
-```
-astro is a command line interface for Airflow working within the Astronomer Cloud or Software
+## Usage
 
-Usage:
   astro [command]
 
-Available Commands:
-  completion  generate the autocompletion script for the specified shell
-  config      Manage project configuration
-  context     Manage Astro & Software contexts
-  deploy      Deploy your project to a Deployment on Astro
-  deployment  Manage your Deployments running on Astronomer
-  dev         Run your Astro project locally
-  help        Help about any command
-  login       Log in to Astronomer
-  logout      Log out of Astronomer
-  version     astro version
-  workspace   Manage Astronomer Workspaces
-
-Flags:
-  -h, --help   help for astro
-
-Use "astro [command] --help" for more information about a command.
-```
+### Core commands
+
+  - `login`       Log in to the Astro CLI
+  - `logout`      Log out of the Astro CLI
+  - `dev init`    Initialize an Astro project in an empty local directory
+  - `dev start`   Build your Astro project into a Docker image and spin up a local Docker container for each Airflow component
+  - `dev stop`    Pause all Docker containers running your local Airflow environment
+  - `dev restart` Stop your Airflow environment, rebuild your Astro project into a Docker image, and restart your Airflow environment with the new Docker image
+  - `deploy`      Deploy code to a Deployment on Astro
+  - `deployment`  Manage your Deployments running on Astronomer
+  - `dev`         Run your Astro project locally
+  - `help`        Help about any Astro CLI command
+  - `version`     Show the running version of the Astro CLI
+  - `workspace`   Manage Astronomer Workspaces
+
+For a list of available Astro CLI commands, see the [Astro CLI command reference](https://docs.astronomer.io/astro/cli/reference).
+
+## Install the Astro CLI
 
-## Install `astro`
+Use these instructions to install, upgrade, or uninstall the Astro CLI.
 
-To install and use the Astro CLI on Mac, you must have:
+### Prerequisites
+
+To install the Astro CLI on Mac, you'll need:
+
+- [Homebrew](https://docs.docker.com/get-docker/)
 - [Docker Desktop](https://docs.docker.com/get-docker/) (v18.09 or higher)
 
-To install and use the Astro CLI on Linux, you must have:
-- [Docker Engine](https://docs.docker.com/engine/install/) (v0.13.1 or higher).
+To install the Astro CLI on Windows, you'll need:
+
+- Windows 10 or later
+- [Docker Desktop for Windows](https://docs.docker.com/desktop/install/windows-install/)
+- [Docker Engine 1.13.1 or later](https://docs.docker.com/engine/install/)
```

Comment:
Should this be v0.13.1 ?

Reply:
@kushalmalani the version information is correct:

![image](https://user-images.githubusercontent.com/104205257/199710468-35f9a29c-e061-46d7-a349-8386533ecc18.png)